test/metabase/sync/util_test.clj

```diff
@@ -110,9 +111,9 @@
     (tu/boolean-ids-and-timestamps (dissoc task-history :duration))))
 
 (deftest task-history-test
-  (let [process-name (tu/random-name)
-        step-1-name  (tu/random-name)
-        step-2-name  (tu/random-name)
+  (let [process-name (tu.random/random-name)
+        step-1-name  (tu.random/random-name)
+        step-2-name  (tu.random/random-name)
```

Comment:
Since you've added the `random-***` functions to metabase.test you should use the `metabase.test` namespace when you can. So this should be `(mt/random-name)`